Building Android fails on rn-qr-generator:verifyReleaseResources task

Hi there!

Thanks for doing this project. It's notably faster to render QRCode with it than with other SVG-based solutions I've tried.

I'm trying to add this module to my project and everything worked fine when running iOS' simulator. But when I tried running this on both an emulator and a real device, and the build failed with the message below.

Any tips on how to fix it, or how to debug the issue?

```
> Task :rn-qr-generator:verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':rn-qr-generator:verifyReleaseResources'.
> 1 exception was raised by workers:
  com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  error: resource android:style/TextAppearance.Material.Widget.Button.Borderless.Colored not found.
  error: resource android:style/TextAppearance.Material.Widget.Button.Colored not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:7: error: resource android:attr/colorError not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:11: error: resource android:attr/colorError not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:15: error: style attribute 'android:attr/keyboardNavigationCluster' not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:7: error: resource android:attr/dialogCornerRadius not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:11: error: resource android:attr/dialogCornerRadius not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values/values.xml:2740: error: resource android:attr/fontStyle not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values/values.xml:2741: error: resource android:attr/font not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values/values.xml:2742: error: resource android:attr/fontWeight not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values/values.xml:2743: error: resource android:attr/fontVariationSettings not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values/values.xml:2744: error: resource android:attr/ttcIndex not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values/values.xml:2908: error: resource android:attr/startX not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values/values.xml:2911: error: resource android:attr/startY not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values/values.xml:2914: error: resource android:attr/endX not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values/values.xml:2917: error: resource android:attr/endY not found.
  myapp/node_modules/rn-qr-generator/android/build/intermediates/res/merged/release/values/values.xml:2925: error: resource android:attr/offset not found.
  error: failed linking references.
  


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.5/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 8m 27s
1311 actionable tasks: 1180 executed, 131 up-to-date
error Failed to build the app. Run CLI with --verbose flag for more details.
Error: Command failed: ./gradlew build -x lint
    at makeError (myapp/node_modules/execa/index.js:174:9)
    at Function.module.exports.sync (myapp/node_modules/execa/index.js:338:15)
    at buildApk (myapp/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:190:22)
    at runOnSpecificDevice (myapp/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:171:7)
    at buildAndRun (myapp/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:156:12)
    at then.result (myapp/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:125:12)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,3 +1,6 @@
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
 
 buildscript {
     repositories {
@@ -12,15 +15,14 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
-        versionCode 1
-        versionName "1.0"
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+        targetSdkVersion safeExtGet('targetSdkVersion', 28)
     }
+
     lintOptions {
         abortOnError false
     }
@@ -32,6 +34,6 @@ repositories {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.google.zxing:core:3.3.0'
+    implementation 'com.google.zxing:core:3.3.3'
 }
   

